<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptItem</class>
 <widget class="QWidget" name="ReceiptItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#headerTitle {
background-color: white;
}
QWidget#headerDateTime {
background-color: white;
}
QWidget#footerTotal {
background-color: white;
}
QWidget#footerCustomerService {
background-color: white;
}
QWidget#footerPhone {
background-color: white;
}
QWidget#body {
background-color: white;
}
QWidget#top {
background: url(:/res/receipt_top.png);
background-color: transparent;
}
QWidget#bottom {
background: url(:/res/receipt_bottom.png);
background-color: transparent;
}
QWidget#restaurantName {
font-size: 16px;
color: black;
}
QWidget#GrandTotalHeaderLabel {
font-size: 22px;
color: black;
}
QWidget#grandTotalHeader {
background-color:white
}
QWidget#GrandTotalFooterLabel {
font-size: 16px;
color: black;
}
QWidget#GrandTotalFooterLabel_2 {
font-size: 16px;
color: black;
}
QWidget#GrandTotalThanksLabel {
font-size: 14px;
color: black;
}
QWidget#grandTotalThanks {
background-color:white
}
QWidget#grandTotalFooter {
background-color: white;
}

QWidget#grandTotalDistance {
background-color: white;
}

QWidget#GrandTotalDistanceLabel_2 {
font-size: 14px;
color: black;
}
QWidget#GrandTotalDistanceLabel {
font-size: 14px;
color: black;
}
QWidget#time {
font-size: 12px;
color: black;
}

QWidget#date {
font-size: 12px;
color: black;
}
QWidget#customerServiceLabel {
font-size: 14px;
color: black;
}
QWidget#customerServiceLabel_2 {
font-size: 10px;
color: black;
}
QWidget#total {
font-size: 12px;
color: black;
}
QWidget#totalLabel {
font-size: 12px;
color: black;
}
QWidget#customerServiceLabel {
font-weight:500;
color:black;
}
QWidget#qty {
font-size: 12px;
color: black;
}
QWidget#item {
font-size: 12px;
color: black;
}
QWidget#price {
font-size: 12px;
color: black;
}
QWidget#qty_2 {
font-size: 12px;
color: black;
}
QWidget#item_2 {
font-size: 12px;
color: black;
}
QWidget#price_2 {
font-size: 12px;
color: black;
}
QWidget#bodyHeader  {
background-color: white;
}</string>
  </property>
  <widget class="QStackedWidget" name="receiptStack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>10</number>
   </property>
   <widget class="QWidget" name="grandTotalHeader">
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>10</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="GrandTotalHeaderLabel">
        <property name="text">
         <string>Grand Total</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="grandTotalFooter">
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QLabel" name="GrandTotalFooterLabel_2">
        <property name="text">
         <string>$00.00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="GrandTotalFooterLabel">
        <property name="text">
         <string>Grand Total</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="grandTotalThanks">
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="GrandTotalThanksLabel">
        <property name="text">
         <string>Thank You ! Have a nice day :)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="grandTotalDistance">
    <widget class="QWidget" name="gridLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="GrandTotalDistanceLabel">
        <property name="text">
         <string>Total Distance Traveled:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="GrandTotalDistanceLabel_2">
        <property name="text">
         <string>miles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="headerTitle">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="restaurantName">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="headerDateTime">
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="date">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="time">
        <property name="text">
         <string>Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="bodyHeader">
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QLabel" name="qty_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Qty </string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="price_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Price</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="item_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Item                                        </string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="body">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLabel" name="item">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>item</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="qty">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>qty</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="price">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>price</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="footerTotal">
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLabel" name="total">
        <property name="text">
         <string>$00.00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="totalLabel">
        <property name="text">
         <string>Total: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="footerCustomerService">
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>10</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="customerServiceLabel">
        <property name="text">
         <string>How was your visit?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="footerPhone">
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>10</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="customerServiceLabel_2">
        <property name="text">
         <string>Call us at  1-Stadium-bros</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="top"/>
   <widget class="QWidget" name="bottom"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
